enforcer-parent/enforcer/src/main/resources/Dockerfile

```diff
@@ -15,9 +15,9 @@
 # -----------------------------------------------------------------------
 
 FROM adoptopenjdk/openjdk11:jre-11.0.13_8-alpine
-RUN apk update && apk upgrade --no-cache
+LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>"
```

Comment:
Shall we use wso2.com as the Maintainers.

Reply:
Let's change this in all the places

Reply:
Updated with https://github.com/wso2/product-microgateway/pull/2497/commits/e46f1559aaad908557b6ce0d62ea4c564beb2173